aspects/aspects.go

```diff
@@ -331,6 +331,42 @@ func (a *Aspect) Set(databag DataBag, request string, value interface{}) error {
 	return notFoundErrorFrom(a, request, "no matching write rule")
 }
 
+// namespaceResult creates a nested namespace around the result that corresponds
+// to the unmatched entry parts. Unmatched placeholders are filled in using maps
+// of all the matching values in the databag.
+func namespaceResult(res interface{}, suffixParts []string) (interface{}, error) {
+	if len(suffixParts) == 0 {
+		return res, nil
+	}
+
+	part := suffixParts[0]
+	if part[0] == '{' && part[len(part)-1] == '}' {
```

Comment:
Side note: maybe a comment saying that the if condition is checking if the part is a placeholder? I wasn't sure what it was doing at first.

Reply:
I've added a couple of lines here, thanks